src/ripple/app/ledger/Ledger.cpp

```diff
@@ -1104,7 +1109,8 @@ bool pendSaveValidated (
 
     if (! app.pendingSaves().shouldWork (ledger->info().seq, isSynchronous))
     {
-        JLOG (app.journal ("Ledger").debug)
+        auto stream = app.journal ("Ledger").debug();
+        JLOG (stream)
```

Comment:
I tried to modify `JLOG` so we could write `JLOG(app.journal("Ledger").debug())` and only evaluate the expression once. I ran into trouble assigning to a variable in the conditional and inverting the conditional. It's OK as is.